SCRWebService/message_handling/tests/test_message_forwarder.py

```diff
@@ -0,0 +1,38 @@
+"""Module that relates to the general processing of an inbound message"""
+import unittest
+from unittest import mock
+from message_handling import message_forwarder as mh
+from builder.pystache_message_builder import MessageGenerationError
+
+
+class TestMessageForwarder(unittest.TestCase):
+    """Tests associated with the MessageForwarder class"""
+
+    def test_forwarding_message_attempts_to_populate_message_template(self):
+        template_mock = mock.MagicMock()
+        interactions_map = {'interaction': template_mock}
+
+        handler = mh.MessageForwarder(interactions_map)
+        input_json = {'check': 'one'}
+        handler.forward_message_to_mhs('interaction', input_json)
+        template_mock.populate_template.assert_called_with(input_json)
+
+    def test_exception_raised_during_population_is_raised(self):
```

Comment:
test name doesnt make sense, why is the exception raise ? what is the behaviour desired?

Reply:
Updated it, think the intention is clearer now but might want double checking it makes sense